2024/FOSDEM/osf_on_amd_5th.md

```diff
@@ -0,0 +1,368 @@
+    * KGPE-D16 needs some love and attention, which, unfortunately, 3mdeb
+      can't humbly provide right now without any support from community
+- Marty Plummer ("hanetzer") is working on adapting Picasso/Cezanne AMD FSP for
+  on a non-Chromebook device ASRock x370 Killer SLI
+    * [Dasharo vPub 0x8 recording](https://www.youtube.com/watch?v=LYPH2Oc6xqU&list=PLuISieMwVBpKevqcC9qEav9ZnCTwkZVW2&index=8&t=393s)
+    * [Dasharo vPub 0x9 recording](https://www.youtube.com/watch?v=gAZw0fTKdYg)
+    * Join [Dasharo Matrix Space](https://matrix.to/#/#dasharo:matrix.org) or
+      [Dasharo vPubs](https://vpub.dasharo.com/o/1) to know more
+
+.center[.image-20[![](/img/ryzen.jpg)]]
+
+.center.footnote[
+[Ryzen photo](https://www.flickr.com/photos/130561288@N04/51371399197)
+by Fritzchens Fritz, CC0 1.0 Universal Public Domain Dedication
+]
+
+???
+
+We tried to contact Immunefi to redirect funds; redirection was needed because
+the whole plan fell apart when it was realized upstreaming was not feasible.
+Unfortunately, we haven't gotten any response from them so far, so we assume
+that some resources are stalled. We also get medium interest from testers;
+some bugs were reported (maybe labeled to Dasharo issues about that), but no
+developers activity from the community. Also, some configurations and bugs
+were exotic enough. We could not invest in reproduction. We contacted NLNet,
+and they said this is too old hardware to support. We also contacted Vikings,
+offering the Dasharo Revenue Sharing Program, but we didn't get a response.
+
+Also, most of the community seems to be interested in free support of that
+platform, which means someone else would have to have serious business
+interests. The last path we could consider is selling SPI chips with Dasharo
+for KGPE-D16, but it is unlikely to render volumes, which could seriously
+impact development.
+
+The platform is still on Dasharo Roadmap but most likely would be removed.
+
+hanetzer is putting some serous efforts here using SerialICE to trace the
+hardware accesses in FSP and PSP. Observes hangs around accesses to PSP and
+SMU.
+
+- PSP binaries: Picasso's work, but Cezanne (and maybe others) do not work
+  with anything but soldered memory and ChromeOS type devices
+- FSP binaries: hardcoded CPUID checks for the mobile variants, which need
+  patching or a wholly different compilation parameters.
+- AMD OpenSIL would be the way forward here
+
+---
+
+# AMD server status
+
+.left-column45[
+.center[.image-60[![](/img/EpycProcessor.jpg)]]
+
+- PoC for Genoa-based (AMD EPYC 9004) reference board Onyx on
+  [GitHub](https://github.com/openSIL/openSIL)
+- [coreboot source also available and merged to upstream
+  repository]((https://review.coreboot.org/q/topic:%22amd_genoa_opensil%22)
+- UEFI EDK2-based PoC code also available on GitHub:
+    * [opensil-uefi-interface](https://github.com/openSIL/opensil-uefi-interface)
+    * [EDK2 Platforms](https://github.com/openSIL/EDKII-Platform)
```

Comment:
Suggest change to '[EDKII Platform]' to avoid confusion with the tianocore/edk2-platforms repository.

Reply:
Changed to EDKII Platform, as suggested.